### ISSUE TITLE
If referencing widget multiple values is set to != UNLIMITED cardinality.

### ISSUE THREAD

**Issue by dwb17** [KostasBlank/entityreference_add_child_to_parent#3]
If the above is true, the module does not seem to work when the assigned values reach the set Cardinality amount.
The Child entity is created, however not assigned to the Parent.
Saving will not do anything, since we can not exceed the set Cardinality value.
Maybe we can have a option to assign the new value to the top of the list, thus "removing" old values as new get inserted.

**Comment by KostasBlank** [comment]
Good point there. I am not sure though what would be the best way to tackle this situation.

Removing references without user approval seems like a bad idea to me.

Why not just display an error message saying that the child entity was saved but was not assigned to parent since the max cardinality has been reached?

Alternatively we can minimise this case altogether if we do not display an add child link if the cardinality is reached.  The only way to arrive to that situation would be by typing the GET parameters to the URL.
